dist/origin-web-common-services.js

```diff
@@ -3193,6 +3194,9 @@ angular.module('openshiftCommonServices')
                                         });
                               }, function(e) {
                                 context.projectPromise.reject(e);
+                                if ((e.status === 403 || e.status === 404) && _.has(opts, 'skipErrorOnProjNotFound')) {
```

Comment:
Would we be interested in a `service/filter/helper` for the `=== 404` kind of logic? Something like `notFound(e) / forbidden(e)`?  I bet we have this scattered around the code.